dapps/v2Explorer/android/app/src/main/java/com/v2explorer/MainApplication.java

```diff
@@ -24,7 +24,7 @@ protected List<ReactPackage> getPackages() {
           @SuppressWarnings("UnnecessaryLocalVariable")
           List<ReactPackage> packages = new PackageList(this).getPackages();
           // Packages that cannot be autolinked yet can be added manually here, for example:
-          // packages.add(new MyReactNativePackage());
+           packages.add(new CustomPackage());
```

Comment:
NonBlocking / Curious: Never added customPackages myself. Do you have any references I can read / look at for future?

Reply:
Yep! Here's the links:

Android guide: https://reactnative.dev/docs/native-modules-android
iOS guide: https://reactnative.dev/docs/native-modules-ios

When we move all into an SDK, i'll try to migrate the code to Kotlin